styleguide/writing-style.md

```diff
@@ -0,0 +1,183 @@
+
+### Commas
+
+Keep it clean and uncluttered. If the sentence can live without that comma - leave it out.
+
+### Serial commas
+
+Absolutely yes, yes, and yes. Use serial commas _when required_ - typically when listing items in a sentence.
+
+Serial commas are also known as 'Oxford commas'.
+
+**Why?** Note the difference between these two sentences:
+
+1. _I went to the shop with my parents, Fred and Wilma._
+1. _I went to the shop with my parents, Fred, and Wilma._
+
+How many people are being referred to in each of these sentences?
+
+* In sentence 1, in addition to me, there are two more people involved: Fred and Wilma, both of whom happen to be my parents.
+* In sentence 2, in addition to me, there are four more people involved: my parents (whose names are not mentioned), as well as Fred, and Wilma.
+
+Therefore, use serial commas to distinguish individual items in a list from those which are not.
+
+### Active/passive voice
+
+Whenever possible, use active voice, which is generally clearer and simpler.
+
+Conventional/natural English sentence word order is subject > verb > object, where the subject or object could be clauses in their own right.
+
+_Active voice_ typically follows this natural word order and is generally easier for readers to follow. More specifically, with active voice, the subject comes before the main verb/action in a sentence.\
+For example:
+* _The XYZ window_ (subject) _displays_ (verb) _the result._ (object)\
+That is:\
+_The XYZ window displays the result._
+* _Select "Settings" > "YAML Migration"_ (subject clause) _to open_ (action) _the YAML migration settings._ (object)\
+That is:\
+_Select "Settings" > "YAML Migration" to open the YAML migration settings._
+
+Passive voice is when the object of a sentence comes before its (main) verb/action.\
+Following on from the examples above (written in passive voice):
+* _The result_ (object) _is displayed_ (verb) _by the XYZ window._ (subject clause)\
+That is:\
+_The result is displayed by the XYZ window._
+* _The YAML migration settings_ (object) _is opened_ (action) _by selecting "Settings" > "YAML Migration"._ (subject clause)\
+That is:\
+_The YAML migration settings is opened by selecting "Settings" > "YAML Migration"._\
+You could make this sentence sound a little more like it's in the active voice through a little rearrangement:\
+_Open the YAML migration settings by selecting "Settings" > "YAML Migration"._\
+However, the fully active voice version (above) is preferable for instructional step-by-step content like this.
+
+Aim to use active voice in instructional step-by-step content and avoid passive voice.\
+Passive voice is useful when you want to emphasize the object, for example when describing the consequence of an activity in the previous sentence. However, stick to active voice for such sentences unless the object requires emphasis.
```

Comment:
Another use case that might be worth mentioning here is using passive voice to avoid placing blame on the user.

Reply:
Oh OK, let's chat about this as I'd like to clarify this for the style guide.

Reply:
E.g. "The system crashed by the tape being inserted incorrectly" (passive and more polite) versus "You inserted the tape incorrectly leading to the system crashing".

Reply:
Added section about this.